sycl/source/detail/event_impl.cpp

```diff
@@ -407,6 +407,69 @@ event_impl::get_info<info::event::command_execution_status>() {
              : info::event_command_status::complete;
 }
 
+template <>
+typename info::platform::version::return_type
+event_impl::get_backend_info<info::platform::version>() const {
+  if (!MIsContextInitialized) {
+    return "Context not initialized, no backend info available";
+  }
+  if (MContext->getBackend() != backend::opencl) {
+    throw sycl::exception(errc::backend_mismatch,
+                          "the info::platform::version info descriptor can "
+                          "only be queried with an OpenCL backend");
+  }
+  if (QueueImplPtr Queue = MQueue.lock()) {
+    try {
+      return Queue->getDeviceImplPtr()
+          ->get_platform()
+          .get_info<info::platform::version>();
+    } catch (...) {
+      std::rethrow_exception(std::current_exception());
+    }
```

Comment:
Could you please clarify what is the point of doing this?

Reply:
Discussed offline, this is redundant, so needs to be removed. Approving the PR on condition that this will be fixed before merging.